COLLABORATOR_GUIDE.md

```diff
@@ -87,6 +87,25 @@ All pull requests that modify executable code should be subjected to
 continuous integration tests on the
 [project CI server](https://ci.nodejs.org/).
 
+Here are some useful CI jobs:
```

Comment:
Should I make the change now or wait until someone else agrees ? Not sure of the protocol here. @gibfahn ?